orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -49,22 +51,34 @@ class CreateWebhookTask implements RetryableTask {
     Map<String, ?> outputsDeprecated = [deprecationWarning: "All webhook information will be moved beneath the key 'webhook', " +
       "and the keys 'statusCode', 'buildInfo', 'statusEndpoint' and 'error' will be removed. Please migrate today."]
 
-    String url = stage.context.url
-    def method = stage.context.method ? HttpMethod.valueOf(stage.context.method.toString().toUpperCase()) : HttpMethod.POST
-    def payload = stage.context.payload
-    def customHeaders = stage.context.customHeaders
-    boolean waitForCompletion = (stage.context.waitForCompletion as String)?.toBoolean()
+    StageData stageData = stage.mapTo(null, StageData.class)
```

Comment:
The `null` seems off, I don't think that's needed is it?

`StageData stageData = stage.mapTo(StageData)`

Reply:
thanks!